build.gradle

```diff
@@ -93,6 +99,9 @@ subprojects {
             opensaml("com.google.guava:guava:24.1.1-jre") {
                 because "opensaml:3.4.0 has a dep on guava:20.0.0 which is vulnerable to CVE-2018-10237"
             }
+            opensaml("org.dom4j:dom4j:2.1.1") {
```

Comment:
Should I have taken this out do you think?  It's kind of redundant with the catch-all substitution?

Reply:
Yes, take it out please! 👍